.github/workflows/release.yml

```diff
@@ -0,0 +1,40 @@
+name: Release
+on:
+  push:
+    tags:
+      - v*
+
+env:
+  ELIXIR_OPTS: "--warnings-as-errors"
+  ERLC_OPTS: "warnings_as_errors"
+  LANG: C.UTF-8
+
+jobs:
+  building_release:
+    strategy:
+      fail-fast: true
+      matrix:
+        otp_release: ["OTP-24.0"]
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 50
+      - name: Install Erlang/OTP
+        run: |
+          cd $RUNNER_TEMP
+          wget -O otp.tar.gz https://repo.hex.pm/builds/otp/ubuntu-18.04/${{ matrix.otp_release }}.tar.gz
+          mkdir -p otp
+          tar zxf otp.tar.gz -C otp --strip-components=1
+          otp/Install -minimal $(pwd)/otp
+          echo "$(pwd)/otp/bin" >> $GITHUB_PATH
```

Comment:
I think it is totally fine to manually download and install OTP however another option worth mentioning is to simply:

```yaml
      - uses: erlef/setup-beam@v1
        with:
          otp-version: ${{ matrix.otp_release }}
```

Reply:
It could be simpler if we can use `setup-beam`.  :)